### ISSUE TITLE
Upload error

### ISSUE THREAD

**Issue by JBel38** [ajinabraham/Mobile-Security-Framework-MobSF#257]
I get this error whenever i attempt to upload an APK file. 

Not sure if i botched my windows install or if something else, my linux install runs perfectly. 
```
System check identified no issues (0 silenced).
October 31, 2016 - 17:10:41
Django version 1.10b1, using settings 'MobSF.settings'
Starting development server at http://127.0.0.1:8000/
Quit the server with CTRL-BREAK.
[31/Oct/2016 17:10:46] "GET / HTTP/1.1" 200 7750
[31/Oct/2016 17:10:46] "GET /static/favicon.ico HTTP/1.1" 200 370070

[2016-10-31 17:10:50]
[ERROR] Uploading File: (C:\Users\dbeliveau\Desktop\Mobile-Security-Framework-Mo
bSF-master\MobSF\views.py, LINE 75 "print "[INFO] MIME Type: " + file_type + " F
ILE: " + request.FILES['file'].name"): 'charmap' codec can't encode character u'
\u2013' in position 56: character maps to <undefined>
Internal Server Error: /upload/
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 3
9, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 248, i
n _legacy_get_response
    response = self._get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 197, i
n _get_response
    "returned None instead." % (callback.__module__, view_name)
ValueError: The view MobSF.views.upload didn't return an HttpResponse object. It
 returned None instead.
[31/Oct/2016 17:10:50] "POST /upload/ HTTP/1.1" 500 69368
```

**Comment by TiberiumPY** [comment]
@JBel38 it's because name of the file is not ASCII-only letters

**Comment by JBel38** [comment]
Derp, why didn't i think of that... Thank you!!!

**Comment by TiberiumPY** [comment]
@JBel38 did it helped you?

**Comment by JBel38** [comment]
Yup, all I had to do was change the name of the file to something more acceptable and it worked like a charm! ^_^

**Comment by ajinabraham** [comment]
@JBel38 Can you share the problematic file name that caused this issue. I am sure other people might also faced this issue.

**Comment by JBel38** [comment]
Absolutely:
Zazzle – Create Custom Gifts_v1.5.5_apkpure.com.apk

**Comment by TiberiumPY** [comment]
@JBel38 maybe this symbol "–"

**Comment by ajinabraham** [comment]
I just got it working fine in Mac. 

```
[INFO] MIME Type: application/octet-stream FILE: Zazzle – Create Custom Gifts_v1.5.5_apkpure.com.apk

[INFO] Performing Static Analysis of Android APK
[01/Nov/2016 18:48:57] "POST /upload/ HTTP/1.1" 200 179
[INFO] Starting Analysis on : Zazzle – Create Custom Gifts_v1.5.5_apkpure.com.apk

[INFO] Analysis is already Done. Fetching data from the DB...
[01/Nov/2016 18:48:57] "GET /StaticAnalyzer/?name=Zazzle%20%E2%80%93%20Create%20Custom%20Gifts_v1.5.5_apkpure.com.apk&type=apk&checksum=feb48768671294a36e009315ed370a2f HTTP/1.1" 200 119521
```

Older version of python 2.7 had shown some issues with unicode before.

**Comment by JBel38** [comment]
Yes, it's definitely the dash/hyphen "-", underscores work fine.

Funny enough, I also used python 3.5.2, as I saw you mention that python 2.7.x has issues with unicode, but to no avail.

**Comment by ajinabraham** [comment]
Well actually I meant the latest python version available in Python 2.7.X branch and not Python3

**Comment by TiberiumPY** [comment]
@JBel38 this program is python-2 only right now
